xarray/core/combine.py

```diff
@@ -520,9 +519,33 @@ def combine_by_coords(
           of all non-null values.
         - 'override': skip comparing and pick variable from first dataset
     data_vars : {'minimal', 'different', 'all' or list of str}, optional
-        Details are in the documentation of concat
+        These data variables will be concatenated together:
+
+        * 'minimal': Only data variables in which the dimension already
+          appears are included.
+        * 'different': Data variables which are not equal (ignoring
+          attributes) across all datasets are also concatenated (as well as
+          all for which dimension already appears). Beware: this option may
+          load the data payload of data variables into memory if they are not
+          already loaded.
+        * 'all': All data variables will be concatenated.
+        * list of str: The listed data variables will be concatenated, in
+          addition to the 'minimal' data variables.
+        If objects are DataArrays, `data_vars` must be 'all'.
     coords : {'minimal', 'different', 'all' or list of str}, optional
-        Details are in the documentation of concat
+        These coordinate variables will be concatenated together:
+
+        * 'minimal': Only coordinates in which the dimension already appears
+          are included.
+        * 'different': Coordinates which are not equal (ignoring attributes)
+          across all datasets are also concatenated (as well as all for which
+          dimension already appears). Beware: this option may load the data
+          payload of coordinate variables into memory if they are not already
+          loaded.
+        * 'all': All coordinate variables will be concatenated, except
+          those corresponding to other dimensions.
+        * list of str: The listed coordinate variables will be concatenated,
+          in addition to the 'minimal' coordinates.
```

Comment:
What do we think about copying and pasting within the same docstring?

I naturally go to try and find whether there are differences; here they're basically the same I think? So another option would be to have `As per the 'data_vars' kwarg, for coords`. 

TBC, either way is much better than the current reference to another docstring. And without these PRs, we wouldn't be able to come to these decisions, so thanks @andersy005

Reply:
@max-sixty, 

I concur. It makes to avoid copying and pasting same docstring for this particular case. 

Are you suggesting something along the lines?

```python
data_vars : {'minimal', 'different', 'all' or list of str}, optional
        These data variables will be concatenated together:

        * 'minimal': Only data variables in which the dimension already
          appears are included.
        * 'different': Data variables which are not equal (ignoring
          attributes) across all datasets are also concatenated (as well as
          all for which dimension already appears). Beware: this option may
          load the data payload of data variables into memory if they are not
          already loaded.
        * 'all': All data variables will be concatenated.
        * list of str: The listed data variables will be concatenated, in
          addition to the 'minimal' data variables.
        If objects are DataArrays, `data_vars` must be 'all'.
    coords : {'minimal', 'different', 'all' or list of str}, optional
       As per the 'data_vars' kwarg, for coords.
```

Reply:
Yes let's do this